board/riscv64/visionfive2/rootfs/etc/rauc/system.conf

```diff
@@ -0,0 +1,18 @@
+[system]
+compatible=infix-riscv64
+bootloader=uboot
+statusfile=/mnt/aux/rauc.status
+mountprefix=/var/lib/rauc/mnt
+bundle-formats=-plain
+max-bundle-download-size=1073741824
+
+[keyring]
+directory=/etc/rauc/keys
+
+[slot.rootfs.0]
+device=/dev/mmcblk1p3
+bootname=primary
```

Comment:
If you change the partition named `rootfs` in `genimage.cfg` to `primary`, can't you use the same device path as on `aarch64` here?

Reply:
Reworking the entire thing right now. Only reason for keeping it named `rootfs` as above was to have the image be loaded from the default U-Boot that comes preloaded on the QSPI flash.

Will make a follow-up commit to address this, thanks!